src/dashboard/cd-dashboard.vue

```diff
@@ -7,34 +7,77 @@
         <dashboard-news />
       </div>
       <div class="cd-dashboard__right-column">
-        <dashboard-children/>
-        <dashboard-stats />
+        <dashboard-children v-if="userProfile && childrenAreVisible" :user-profile="userProfile"/>
+        <dashboard-stats v-if="userDojos && statsAreVisible" :user-dojos="filterUserDojos('champion')"/>
       </div>
     </div>
   </div>
 </template>
 
 <script>
-import DashboardEvents from '@/dashboard/cd-dashboard-events';
-import DashboardProjects from '@/dashboard/cd-dashboard-projects';
-import DashboardChildren from '@/dashboard/cd-dashboard-children';
-import DashboardNews from '@/dashboard/cd-dashboard-news';
-import DashboardStats from '@/dashboard/cd-dashboard-stats';
+  import { mapGetters } from 'vuex';
+  import DojoService from '@/dojos/service';
+  import UserService from '@/users/service';
+  import DashboardEvents from '@/dashboard/cd-dashboard-events';
+  import DashboardProjects from '@/dashboard/cd-dashboard-projects';
+  import DashboardChildren from '@/dashboard/cd-dashboard-children';
+  import DashboardNews from '@/dashboard/cd-dashboard-news';
+  import DashboardStats from '@/dashboard/cd-dashboard-stats';
 
-export default {
-  name: 'Home',
-  data() {
-    return {
-    };
-  },
-  components: {
-    DashboardEvents,
-    DashboardProjects,
-    DashboardChildren,
-    DashboardNews,
-    DashboardStats,
-  },
-};
+  export default {
+    name: 'Home',
+    data() {
+      return {
+        userDojos: null,
+        userProfile: null,
+      };
+    },
+    components: {
+      DashboardEvents,
+      DashboardProjects,
+      DashboardChildren,
+      DashboardNews,
+      DashboardStats,
+    },
+    computed: {
+      ...mapGetters(['loggedInUser']),
+      statsAreVisible() {
+        return this.filterUserDojos('champion').length > 0;
+      },
+      childrenAreVisible() {
+        return !!(this.userProfile &&
+          this.userProfile.children &&
+          this.userProfile.children.length > 0);
+      },
+      highestUserRole() {
+        /* eslint-disable no-nested-ternary */
+        return this.filterUserDojos('champion').length ? 'champion' :
+          this.filterUserDojos('mentor').length ? 'mentor' :
+            this.filterUserDojos('parent-guardian').length ? 'parent-guardian' :
+              'ninja';
+        /* eslint-enable no-nested-ternary */
+      },
+    },
+    methods: {
+      filterUserDojos(role) {
+        return this.userDojos.filter(ud => ud.userTypes.includes(role));
```

Comment:
`includes` isn't supported in IE11 if we still care...

Reply:
5% of traffic from IE 11 :/
Won't babel transform it into something usable, like any es6 function ?

Reply:
Babel only transforms syntax, not methods. You'd need to polyfill, unfortunately. Or implement it in a different way.

Reply:
I mean, filter isn't supported either then so.. well.. too late ? :dagger:

Reply:
Filter is supported since IE9... https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter#Browser_compatibility

Reply:
whoops, typed filter in caniuse and it's the CSS attribute, not the js one :(
Apparently we already use includes in FTB, so.. Well, farewell IE11

Reply:
I did the same thing originally. 😅

Or we remove the usage of `includes` in FTB..? 🤔 Should probably do a general test of IE11 sometime soon...